environment.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
   - pip:
     - nbdev
     - tqdm
+    - plotly-resampler
```

Comment:
Could you please include `plotly-resampler` as a conda dependency instead of `pip`? Thank you!

Reply:
I am fine with adding the [conda-forge feedstock](https://github.com/conda-forge/plotly-resampler-feedstock) of plotly-resampler.
However, I initially added the `pip` dependency, as 
* (@jonasvdd and me) are the maintainers of the pypi deploy - which is not the case the conda-forge deploy
* the conda-forge deploy gets updated less frequent (is still 2 versions behind the current pypi version)

So this makes me curious why this makes you prefer `conda` over `pip`? :eyes: 

P.S.: I have little to no experience with conda (for this project I created my very first conda env :upside_down_face: )

Reply:
hey @jvdd! The main reason behind adding the library to conda is just as a sanity check. Since we are maintaining the [conda-forge/statsforecast](https://github.com/conda-forge/statsforecast-feedstock) repo, we must add the new dependency to the [recipe](https://github.com/conda-forge/statsforecast-feedstock/blob/main/recipe/meta.yaml) file to prevent errors or conflicts building the library. 

The alternative that comes to my mind (since you're not maintaining the conda-forge/plotly-resampler) is to try to import the library only when `engine=="plotly-resampler"` and throw an error if it is not installed.  With that change, we can include `plotly-resampler` as a core dependency in pip even if we don't include it as part of conda-forge/statsforecast. 

I'll be glad to read your thoughts on that.

Reply:
I like the alternative you suggested. I've made the necessary updates to the PR based on your suggestion.

As for the conda-forge/plotly-resampler package, I plan to become a maintainer in the future. Once I'm a maintainer, we can consider updating the dependency to conda-forge (as there will be more guarantees about the maintenance etc.).

Thanks for your feedback and suggestion!